www/src/pages/en/usage/first-steps.md

```diff
@@ -23,7 +23,8 @@ Of course, if you prefer to use a different auth provider, you can also use one
 5. Click "Reset Secret", copy the new secret, and add it to your `.env` as `DISCORD_CLIENT_SECRET`.
 6. Click "Add Redirect" and type in `http://localhost:3000/api/auth/callback/discord`.
    - For production deployment, follow the previous steps to create another Discord Application, but this time replace `http://localhost:3000` with the URL that you are deploying to.
-7. Save Changes. You should now be able to log in.
+7. Save Changes.
+8. Set the NEXTAUTH_SECRET as some random string too. You should now be able to log in.
```

Comment:
```suggestion
8. Set the `NEXTAUTH_SECRET` in `.env`. In development any string will work, for production see the note in `.env` on generating a secure secret.

You should now be able to log in.
```

Reply:
also is the secret even required in dev? i thought it just complains in the console if you don't have it?

Reply:
> If no value is specified in development (and there is no NEXTAUTH_SECRET variable either), it uses a hash for all configuration options, including OAuth Client ID / Secrets for entropy.

https://next-auth.js.org/warnings#no_secret

Reply:
hmm probably better to suggest adding it

Reply:
@c-ehrlich Much better wording, thanks! :)